src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_log_analytics_workspace.py

```diff
@@ -478,3 +478,36 @@ def test_monitor_log_analytics_workspace_data_export(self, resource_group, accou
         self.cmd('monitor log-analytics workspace data-export delete -g {rg} --workspace-name {workspace_name} -n {data_export_name} -y')
         with self.assertRaisesRegex(SystemExit, '3'):
             self.cmd('monitor log-analytics workspace data-export show -g {rg} --workspace-name {workspace_name} -n {data_export_name}')
+
+    @ResourceGroupPreparer(name_prefix='cli_test_monitor_workspace_table', location='WestEurope')
+    @AllowLargeResponse()
+    def test_monitor_log_analytics_workspace_table(self, resource_group):
```

Comment:
Add test for `az monitor log-analytics workspace update` with the argument `--defaultDataCollectionRuleResourceId` as well.